kv/tikv/transaction/commands4a_test.go

```diff
@@ -0,0 +1,134 @@
+package transaction
+
+import (
+	"testing"
+
+	"github.com/pingcap-incubator/tinykv/kv/util/engine_util"
+	"github.com/pingcap-incubator/tinykv/proto/pkg/kvrpcpb"
+	"github.com/stretchr/testify/assert"
+)
+
+// TestGetValue4A getting a value works in the simple case.
+func TestGetValue4A(t *testing.T) {
```

Comment:
why these are empty?

Reply:
I thought we wanted the students to fill these out? I can fill them out with the actual tests if we want.

Reply:
students don't need to fill the test, all tests are provided by us to make sure their implementation is correct.